src/byteArrayParser.js

```diff
@@ -5,7 +5,7 @@
 /**
  * Reads a string of 8-bit characters from an array of bytes and advances
  * the position by length bytes.  A null terminator will end the string
- * but will not effect advancement of the position.  Trailing and leading
```

Comment:
Following comment applies to entire change:
Can you please run lint and ensure we are all clear. Looks like you have changed the formatting and this might be hard to track